### PR TITLE
basic custom dataset model

### PR DESCRIPTION
**Pull request by sumitmamtani**

### CODE REVIEW

**Review by YashBit — APPROVED** [review]
@sumitmamtani 

1. Still need to add image pretrained on ImageNet or not
2. Metric Design 
3. Hyperparameter inclusion
